### PR TITLE
Feat: Implement automated updates with Dependabot

### LINKED ISSUE #1438: Configure Automated Dependabot Updates
We need to make sure our code is constantly up to date especially for our `develop` branch. To do this we need to implement automated updates using dependabot. This has already been done with the Talawa repo, but we also need it done for this one.

Requirements

1. Weekly updates
1. Run against the `default` branch
1. Use the appropriate package system
1. Use the appropriate root directory

Other information

1. Documentation: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
1. Talawa Example: https://github.com/PalisadoesFoundation/talawa/blob/develop/.github/dependabot.yaml

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Harsh1s**
**What kind of change does this PR introduce?**

- [ ] Bug Fix
- [ ] Feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Documentation Update

**Issue Number:**

Fixes #1438

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

N/A

**If relevant, did you update the documentation?**

I have been assigned the issue fo the same, I plan on doing it after this PR.

**Summary**

Implement automated updates using Dependabot for weekly updates on the default branch. Configuration includes appropriate package system and root directory settings.

**Does this PR introduce a breaking change?**

No

**Other information**

N/A

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    # Customize commit message prefix
+    commit-message:
+      prefix: "chore(deps):"
+    # Optional parameters
+    # # List of GitHub usernames to assign the PR to
+    # assignees:
+    #   - "username1"
+    #   - "username2"
+    # # Limit the number of open pull requests Dependabot will create
+    # open-pull-requests-limit: 10
```
Comment (EshaanAgg) [inline .github/dependabot.yaml:30]: Please remove the comments for the configuration options that we are not planning to use in the workflow.

**Review by EshaanAgg — CHANGES_REQUESTED** [review]

File: .github/dependabot.yaml (hunk 1 above)
Comment (Harsh1s) [reply .github/dependabot.yaml]: Done!

File: videos/GhXHBQar_Gvideo.mp4
Comment (EshaanAgg) [inline videos/GhXHBQar_Gvideo.mp4:1]: Delete this extraneous file.

**Review by EshaanAgg — CHANGES_REQUESTED** [review]

File: videos/GhXHBQar_Gvideo.mp4
Comment (Harsh1s) [reply videos/GhXHBQar_Gvideo.mp4:1]: Done. Appreciate the patience!

**Comment by EshaanAgg** [comment]
Merging this PR because the workflows are failing due to the incorrect configuration in workflow files and not the changes.

**Pull request merged**